src/ensembl/src/content/app/species/SpeciesPage.tsx

```diff
@@ -39,14 +39,28 @@ type SpeciesPageParams = {
 };
 
 const SpeciesPage = () => {
-  const { changeGenomeId } = useSpeciesRouting();
-
   const { genomeId } = useParams() as SpeciesPageParams;
+  const dispatch = useDispatch();
+
+  const changeGenomeId = useCallback(
+    (genomeId: string) => {
+      const params = {
+        genomeId
+      };
+
+      dispatch(replace(urlFor.speciesPage(params)));
```

Comment:
I think the more react-router-y way would be to add a `<Redirect to.../>` component in App.tsx:

```
  <Route path={`/species/:genomeId`} component={SpeciesPage} />
  <Redirect exact from='/species' to={urlFor.speciesSelector()}/>
```

Reply:
That's done